### ISSUE TITLE
Fix the "io exception" in client validation of 6-node and 15-node performance tests.

### ISSUE THREAD

**Issue by ljianghedera** [hashgraph/hedera-services#767]
**Summary of the defect**
Seeing lots of "io exception" errors in 6-Node and 15-Node JRS performance tests in HAPIClientValidator validation.

**How to reproduce (if possible)**

Check out these slack links: 

6-Node: https://hedera-hashgraph.slack.com/archives/CKWHL8R9A/p1605115960225300

15-Node:  https://hedera-hashgraph.slack.com/archives/CKWHL8R9A/p1605101140223100 

**Service logs (if applicable)**
```
...
```
---- HAPIClientValidator FAILED validation ----
<<INFO>>
ClientNode - Suite                    Result
Node 0 - CryptoCreateForSuiteRunner   PASSED
Node 1 - CryptoCreateForSuiteRunner   PASSED
Node 2 - CryptoCreateForSuiteRunner   PASSED
Node 3 - CryptoCreateForSuiteRunner   PASSED
Node 4 - CryptoCreateForSuiteRunner   PASSED
Node 5 - CryptoCreateForSuiteRunner   PASSED
<<ERROR>>
Exception in thread "thread38" Exception in thread "thread29" Exception in thread "thread3" Exception in thread "thread23" Exception in thread "thread9" Exception in thread "thread25" Exception in thread "thread18" Exception in thread "thread27" Exception in thread "thread6" Exception in thread "thread26" Exception in thread "thread34" Exception in thread "thread11" Exception in thread "thread4" java.lang.AssertionError: Operation 'HapiCryptoTransfer{sigs=1, payer=sender, node=0.0.3, transfers=[0.0.66214 <- +1, 0.0.66211 -> -1], tokenTransfers=}' :: UNAVAILABLE: io exception
Exception in thread "thread5" java.lang.AssertionError: Operation 'HapiCryptoTransfer{sigs=1, payer=sender, node=0.0.3, transfers=[0.0.66214 <- +1, 0.0.66211 -> -1], tokenTransfers=}' :: UNAVAILABLE: io exception
Exception in thread "thread2" Exception in thread "thread37" java.lang.AssertionError: Operation 'HapiCryptoTransfer{sigs=1, payer=sender, node=0.0.3, transfers=[0.0.66214 <- +1, 0.0.66211 -> -1], tokenTransfers=}' :: UNAVAILABLE: io exception
Exception in thread "thread31" Exception in thread "thread10" Exception in thread "thread1" java.lang.AssertionError: Operation 'HapiCryptoTransfer{sigs=1, payer=sender, node=0.0.3, transfers=[0.0.66214 <- +1, 0.0.66211 -> -1], tokenTransfers=}' :: UNAVAILABLE: io exception
Exception in thread "thread17" Exception in thread "thread7" java.lang.AssertionError: Operation 'HapiCryptoTransfer{sigs=1, payer=sender, node=0.0.3, transfers=[0.0.66214 <- +1, 0.0.66211 -> -1], tokenTransfers=}' :: UNAVAILABLE: io exception
Exception in thread "thread12" java.lang.AssertionError: Operation 'HapiCryptoTransfer{sigs=1, payer=sender, node=0.0.3, transfers=[0.0.66214 <- +1, 0.0.66211 -> -1], tokenTransfers=}' :: UNAVAILABLE: io exception
Exception in thread "thread13" java.lang.AssertionError: Operation 'HapiCryptoTransfer{sigs=1, payer=sender, node=0.0.3, transfers=[0.0.66214 <- +1, 0.0.66211 -> -1], tokenTransfers=}' :: UNAVAILABLE: io exception
Exception in thread "thread20" java.lang.AssertionError: Operation 'HapiCryptoTransfer{sigs=1, payer=sender, node=0.0.3, transfers=[0.0.66214 <- +1, 0.0.66211 -> -1], tokenTransfers=}' :: UNAVAILABLE: io exception
Exception in thread "thread36" java.lang.AssertionError: Operation
...
```
**Environment:**
- OS: [e.g. Ubuntu 18.04]
- Java: [e.g. OpenJDK 11.0.4]
- Hedera Services Version: [e.g. 0.0.5]
- HAPI Version: [e.g. 0.0.5]

**Additional Context**
Add any other context about the problem here. Attach any logs here, if applicable.

**Comment by ljianghedera** [comment]
At the end, the test client is getting the (error) messages like these: 

```
2020-11-11 21:29:41.523 INFO   172  RunLoadTest - Thread thread15 final ops 90004 in 2700 seconds, TPS 32.061066 
2020-11-11 21:29:41.525 INFO   172  RunLoadTest - Thread thread28 final ops 90548 in 2700 seconds, TPS 32.276653 
2020-11-11 21:29:41.525 INFO   172  RunLoadTest - Thread thread25 final ops 90331 in 2700 seconds, TPS 27.882036 
2020-11-11 21:29:41.526 INFO   124  RunLoadTest - Total Ops submitted 3600734, actual TPS 1333.6052
2020-11-11 21:29:41.526 INFO   227  HapiApiSpec - 'RunCryptoTransfers' finished initial execution of RunLoadTest{}
2020-11-11 21:29:41.531 INFO   290  HapiApiSpec - 'RunCryptoTransfers' - executed 3600736 ledger ops.
2020-11-11 21:29:41.531 INFO   291  HapiApiSpec - 'RunCryptoTransfers' - now finalizing 78771 more pending ops...
2020-11-11 21:29:53.277 INFO   140  HapiTxnOp - 'RunCryptoTransfers' -  submitting HapiCryptoTransfer{node=0.0.3} via body=transactionID { transactionValidStart { seconds: 1605130133 nanos: 21408347 } accountID { accountNum: 66200 } } nodeAccountID { accountNum: 3 } transactionFee: 408126 transactionValidDuration { seconds: 120 } memo: "\303\203\302\256\303\202\302\267\303\203\302\271tF8\303\202\302\256J\303\203\302\213\303\203\302\220\303\203\302\216" cryptoTransfer { transfers { accountAmounts { accountID { accountNum: 66206 } amount: 900000000000 } accountAmounts { accountID { accountNum: 66200 } amount: -900000000000 } } }; sigs=sigPair { pubKeyPrefix: "\303" ed25519: "\266\205mk\222/\003c\026\301\b9\215\310\006\352\210\364%\rX\364\211\2247A\241\225!X\226\365\315q(\334\324\314O<\213\323\243T\324b\223\267\203\226\3166\204\242\b5\246`\213\004\365\343\370\a" } sigPair { pubKeyPrefix: "R" ed25519: "\357;\2231O\t\373\253 jz\260\341 \271\256\001H\b\216\201\273\005}\256\266\031\332\237\312F\t\026\260\320\223C\\&\271\247d\305U7\205\006FWH.\361\324\000\267\340\350\304p\000\340\317\363\t" }
2020-11-11 21:29:53.277 INFO   140  HapiTxnOp - 'RunCryptoTransfers' -  submitting HapiCryptoTransfer{node=0.0.3} via body=transactionID { transactionValidStart { seconds: 1605130133 nanos: 21408350 } accountID { accountNum: 66200 } } nodeAccountID { accountNum: 3 } transactionFee: 408126 transactionValidDuration { seconds: 120 } memo: "\303\203\302\256\303\202\302\267\303\203\302\271tF8\303\202\302\256J\303\203\302\213\303\203\302\220\303\203\302\216" cryptoTransfer { transfers { accountAmounts { accountID { accountNum: 66206 } amount: 900000000000 } accountAmounts { accountID { accountNum: 66200 } amount: -900000000000 } } }; sigs=sigPair { pubKeyPrefix: "\303" ed25519: "\216R\274-\277\345|\314\030\230+mu~\367\323\316\021\274\375\304\255\"\2343\303B\271\312\220\247\177\017\241\353=U\223\326$\212\017\313\025\204\354\3135\217\355\024D\311\367\355\354\244\003\323\201\371\320v\017" } sigPair { pubKeyPrefix: "R" ed25519: " \207\337B\303\340\330\353:\344\362pY\001#\247\317\005p\3617\327z\344D\261\317\020\374\340@W>7\305{Y\02796d\262\353(V\312\217\2258\273\315\017V\263\275\313\001\360\322\360+\227B\001" }
2020-11-11 21:29:53.277 INFO   140  HapiTxnOp - 'RunCryptoTransfers' -  submitting HapiCryptoTransfer{node=0.0.3} via body=transactionID { transactionValidStart { seconds: 1605130133 nanos: 21408345 } accountID { accountNum: 66200 } } nodeAccountID { accountNum: 3 } transactionFee: 408126 transactionValidDuration { seconds: 120 } memo: "\303\203\302\256\303\202\302\267\303\203\302\271tF8\303\202\302\256J\303\203\302\213\303\203\302\220\303\203\302\216" cryptoTransfer { transfers { accountAmounts { accountID { accountNum: 66206 } amount: 900000000000 } accountAmounts { accountID { accountNum: 66200 } amount: -900000000000 } } }; sigs=sigPair { pubKeyPrefix: "\303" ed25519: "\275\222a\355n\0227K\005\334\346\312\275\0335]\335\024\327\301|\225\323p\300\253\251<\361\265\002E\321\245\017\204\'\351\n.t\341%\353\261{\035k\250\f1\310>q+r(\"\016\025\005\241\355\006" } sigPair { pubKeyPrefix: "R" ed25519: "\370\"\345\37574\244\270\206\t\t\260r\250\317B\220\235//\275\360g\367\300\235\356\335\033A_Y\352R\250\002\254\202>o\356\027l\321\231\345jx\253\210\262m\335\225\306\033}\016/\033W\324\211\t" }
2020-11-11 21:29:53.277 INFO   140  HapiTxnOp - 'RunCryptoTransfers' -  submitting HapiCryptoTransfer{node=0.0.3} via body=transactionID { transactionValidStart { seconds: 1605130133 nanos: 21408351 } accountID { accountNum: 66200 } } nodeAccountID { accountNum: 3 } transactionFee: 408126 transactionValidDuration { seconds: 120 } memo: "\303\203\302\256\303\202\302\267\303\203\302\271tF8\303\202\302\256J\303\203\302\213\303\203\302\220\303\203\302\216" cryptoTransfer { transfers { accountAmounts { accountID { accountNum: 66206 } amount: 900000000000 } accountAmounts { accountID { accountNum: 66200 } amount: -900000000000 } } }; sigs=sigPair { pubKeyPrefix: "\303" ed25519: "\221\325+m\317\225o\220\371\312\252\360\250k\366\\F\b\020\'\346\017}\0007J@o\001Y\t\017\305{\325\310\356\024]V\202\201a\206\330]\201]_{%\367\017UmXj0\205\337}\321\311\a" } sigPair { pubKeyPrefix: "R" ed25519: "\n\342dR\242t\361\224$9\244@\351\347c|\353\265\b[o\3423t4\264\f\254\375\375\a\361\220\270\367w\232\251\270br\265\031c\031\004\233\247W\341w\206\217\201\025\371\307\222\237\207J\240\003\v" }
2020-11-11 21:29:53.277 INFO   140  HapiTxnOp - 'RunCryptoTransfers' -  submitting HapiCryptoTransfer{node=0.0.3} via body=transactionID { transactionValidStart { seconds: 1605130133 nanos: 21408348 } accountID { accountNum: 66200 } } nodeAccountID { accountNum: 3 } transactionFee: 408126 transactionValidDuration { seconds: 120 } memo: "\303\203\302\256\303\202\302\267\303\203\302\271tF8\303\202\302\256J\303\203\302\213\303\203\302\220\303\203\302\216" cryptoTransfer { transfers { accountAmounts { accountID { accountNum: 66206 } amount: 900000000000 } accountAmounts { accountID { accountNum: 66200 } amount: -900000000000 } } }; sigs=sigPair { pubKeyPrefix: "\303" ed25519: "\343\360P\331\224\0229\331i\264\227\031I@\340@\256&\245\325\214\235D\276\237\0351\310\226\021Y>J9\352\000\276\001\234)exVH)\265<\236,\205\031:\207\365\236\021\321\206\235\240\\\247O\004" } sigPair { pubKeyPrefix: "R" ed25519: "x\025\030[\t\254\275\376%C\177\357A\006\314\372I\233\205\210\205\301.\363\t\241\334\016Rt\271?\344zx$<9\371;\262\027\305;\254\330(\202\\u\217\354\023\336b\362\352_\237\267\306\025\334\016" }
2020-11-11 21:29:53.277 INFO   140  HapiTxnOp - 'RunCryptoTransfers' -  submitting HapiCryptoTransfer{node=0.0.3} via body=transactionID { transactionValidStart { seconds: 1605130133 nanos: 21408344 } accountID { accountNum: 66200 } } nodeAccountID { accountNum: 3 } transactionFee: 408126 transactionValidDuration { seconds: 120 } memo: "\303\203\302\256\303\202\302\267\303\203\302\271tF8\303\202\302\256J\303\203\302\213\303\203\302\220\303\203\302\216" cryptoTransfer { transfers { accountAmounts { accountID { accountNum: 66206 } amount: 900000000000 } accountAmounts { accountID { accountNum: 66200 } amount: -900000000000 } } }; sigs=sigPair { pubKeyPrefix: "\303" ed25519: "\333\371\031\n\356A\022\215\254P\321\350\t\213\346}\354>\2754TE\304\304\335\307\320+t\244J\221\245\020\031\274=\367\023\370\252\374x+u\223\213Q?\026\0063z\3737g\bqY\233t\223E\001" } sigPair { pubKeyPrefix: "R" ed25519: "\024\257\201x\226^\262L\023\320\205\231k\v\212\002\213\377aL\324\321\222\362\320\266d\322I\362ap\377[\035\v\2147fh@\360\333!\272\253b\233\002\b\207/\327j\200\372nc\3270^Hh\a" }
2020-11-11 21:29:53.277 INFO   140  HapiTxnOp - 'RunCryptoTransfers' -  submitting HapiCryptoTransfer{node=0.0.3} via body=transactionID { transactionValidStart { seconds: 1605130133 nanos: 21408346 } accountID { accountNum: 66200 } } nodeAccountID { accountNum: 3 } transactionFee: 408126 transactionValidDuration { seconds: 120 } memo: "\303\203\302\256\303\202\302\267\303\203\302\271tF8\303\202\302\256J\303\203\302\213\303\203\302\220\303\203\302\216" cryptoTransfer { transfers { accountAmounts { accountID { accountNum: 66206 } amount: 900000000000 } accountAmounts { accountID { accountNum: 66200 } amount: -900000000000 } } }; sigs=sigPair { pubKeyPrefix: "\303" ed25519: "C\331\vP\310\221\307\261\214\301L\026\252\306\032>\300\033B\364\312\026`\303\023y\261\t3\255\020\333\353\207%7\355nO\207\243\374\376\303*\341\"\334\313s\'jJr9cJ\314W+{\277\002\000" } sigPair { pubKeyPrefix: "R" ed25519: "\366V\273\'\332\0053\354\315\223S0\301x\"Ry\303\034\"\271\021\344\245\333\226f\275&\306Lb\020\333n\374\a\032D\254S\004\vyi\254\245\265\fd\200\344\377\236\032\3130\200\301\351f\243`\005" }
2020-11-11 21:29:53.277 INFO   140  HapiTxnOp - 'RunCryptoTransfers' -  submitting HapiCryptoTransfer{node=0.0.3} via body=transactionID { transactionValidStart { seconds: 1605130133 nanos: 21408349 } accountID { accountNum: 66200 } } nodeAccountID { accountNum: 3 } transactionFee: 408126 transactionValidDuration { seconds: 120 } memo: "\303\203\302\256\303\202\302\267\303\203\302\271tF8\303\202\302\256J\303\203\302\213\303\203\302\220\303\203\302\216" cryptoTransfer { transfers { accountAmounts { accountID { accountNum: 66206 } amount: 900000000000 } accountAmounts { accountID { accountNum: 66200 } amount: -900000000000 } } }; sigs=sigPair { pubKeyPrefix: "\303" ed25519: "h\253s\324\326\201\371\320-\263@\3179\363\002\nqAy\002G\37369|\362\337\253g\323\276_\200\250Z\f\240~\213\346\023\3771\260\317X\325\210r!r\aZ\363\204\203\203H\366Q<\254\317\v" } sigPair { pubKeyPrefix: "R" ed25519: "DvX\332\347\277\326V\206\333h\305^\033\3638N\331KS\366f\221p/\256e2\037\331\331\002\'\370\317\265\307\271\037\326\247\002\272\354\312\n\337\206\361\024\266a\365\220\211[\021\256\021x\270@\236\t" }
2020-11-11 21:30:23.384 WARN   199  HapiSpecOperation - 'RunCryptoTransfers' - HapiCryptoTransfer{sigs=2, node=0.0.3, transfers=[0.0.66206 <- +900000000000, 0.0.66200 -> -900000000000], tokenTransfers=} failed!
java.lang.AssertionError: Wrong status! expected:<SUCCESS> but was:<UNKNOWN>
	at org.junit.Assert.fail(Assert.java:88) ~[SuiteRunner.jar:?]
	at org.junit.Assert.failNotEquals(Assert.java:834) ~[SuiteRunner.jar:?]
	at org.junit.Assert.assertEquals(Assert.java:118) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.transactions.HapiTxnOp.resolveStatus(HapiTxnOp.java:251) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.transactions.HapiTxnOp.submitOp(HapiTxnOp.java:209) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.HapiSpecOperation.execFor(HapiSpecOperation.java:179) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.utilops.CustomSpecAssert.allRunFor(CustomSpecAssert.java:38) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.utilops.CustomSpecAssert.allRunFor(CustomSpecAssert.java:47) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.transactions.HapiTxnOp.rechargePayerFor(HapiTxnOp.java:264) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.transactions.HapiTxnOp.resolveStatus(HapiTxnOp.java:236) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.transactions.HapiTxnOp.finalizeExecFor(HapiTxnOp.java:331) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.infrastructure.DelegatingOpFinisher.finishFor(DelegatingOpFinisher.java:41) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.HapiApiSpec.lambda$startFinalizingOps$3(HapiApiSpec.java:266) ~[SuiteRunner.jar:?]
	at java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1807) [?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) [?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) [?:?]
	at java.lang.Thread.run(Thread.java:835) [?:?]
2020-11-11 21:30:23.384 WARN   199  HapiSpecOperation - 'RunCryptoTransfers' - HapiCryptoTransfer{sigs=2, node=0.0.3, transfers=[0.0.66206 <- +900000000000, 0.0.66200 -> -900000000000], tokenTransfers=} failed!
java.lang.AssertionError: Wrong status! expected:<SUCCESS> but was:<UNKNOWN>
	at org.junit.Assert.fail(Assert.java:88) ~[SuiteRunner.jar:?]
	at org.junit.Assert.failNotEquals(Assert.java:834) ~[SuiteRunner.jar:?]
	at org.junit.Assert.assertEquals(Assert.java:118) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.transactions.HapiTxnOp.resolveStatus(HapiTxnOp.java:251) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.transactions.HapiTxnOp.submitOp(HapiTxnOp.java:209) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.HapiSpecOperation.execFor(HapiSpecOperation.java:179) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.utilops.CustomSpecAssert.allRunFor(CustomSpecAssert.java:38) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.utilops.CustomSpecAssert.allRunFor(CustomSpecAssert.java:47) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.transactions.HapiTxnOp.rechargePayerFor(HapiTxnOp.java:264) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.transactions.HapiTxnOp.resolveStatus(HapiTxnOp.java:236) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.transactions.HapiTxnOp.finalizeExecFor(HapiTxnOp.java:331) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.infrastructure.DelegatingOpFinisher.finishFor(DelegatingOpFinisher.java:41) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.HapiApiSpec.lambda$startFinalizingOps$3(HapiApiSpec.java:266) ~[SuiteRunner.jar:?]
	at java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1807) [?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) [?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) [?:?]
	at java.lang.Thread.run(Thread.java:835) [?:?]
2020-11-11 21:30:23.384 WARN   199  HapiSpecOperation - 'RunCryptoTransfers' - HapiCryptoTransfer{sigs=2, node=0.0.3, transfers=[0.0.66206 <- +900000000000, 0.0.66200 -> -900000000000], tokenTransfers=} failed!
java.lang.AssertionError: Wrong status! expected:<SUCCESS> but was:<UNKNOWN>
	at org.junit.Assert.fail(Assert.java:88) ~[SuiteRunner.jar:?]
	at org.junit.Assert.failNotEquals(Assert.java:834) ~[SuiteRunner.jar:?]
	at org.junit.Assert.assertEquals(Assert.java:118) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.transactions.HapiTxnOp.resolveStatus(HapiTxnOp.java:251) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.transactions.HapiTxnOp.submitOp(HapiTxnOp.java:209) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.HapiSpecOperation.execFor(HapiSpecOperation.java:179) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.utilops.CustomSpecAssert.allRunFor(CustomSpecAssert.java:38) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.utilops.CustomSpecAssert.allRunFor(CustomSpecAssert.java:47) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.transactions.HapiTxnOp.rechargePayerFor(HapiTxnOp.java:264) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.transactions.HapiTxnOp.resolveStatus(HapiTxnOp.java:236) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.transactions.HapiTxnOp.finalizeExecFor(HapiTxnOp.java:331) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.infrastructure.DelegatingOpFinisher.finishFor(DelegatingOpFinisher.java:41) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.HapiApiSpec.lambda$startFinalizingOps$3(HapiApiSpec.java:266) ~[SuiteRunner.jar:?]
	at java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1807) [?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) [?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) [?:?]
	at java.lang.Thread.run(Thread.java:835) [?:?]
...
```

This is interesting.

**Comment by JeffreyDallas** [comment]
There is no log uploaded to AWS.
But my guess is 1) Server java process being killed before client finished, so lead to io exception 
2) "Wrong status! expected:<SUCCESS> but was:<UNKNOWN>" was due to too many request were queued inside pipeline, may need lower TPS or use a longer waiting time in ci map settings.

**Comment by ljianghedera** [comment]
> There is no log uploaded to AWS.

The log files are still uploaded. It's just the slack link somehow doesn't point to the exact location anymore. Here is the link to client log file: https://s3.console.aws.amazon.com/s3/object/hedera-service-regression-jrs?region=us-east-2&prefix=15N_15C/20201111-1212-AWS-Weekly-Services-Crypto-Restart-Performance-15N-15C/HCS-Restart-Performance-10k-62m/node0000-TestClient/hapi-client-combined.log. 

> But my guess is 1) Server java process being killed before client finished, so lead to io exception

Yeah, I think that's what's happening. But should this not happen anyway? Or do we think it's a acceptable error due to the long running period.

> 2) "Wrong status! expected: but was:" was due to too many request were queued inside pipeline, may need lower TPS or use a longer waiting time in ci map settings.

This is actually the case. And from the stats diagrams, we can observe the trans/sec has dram fluctuations. I will mention this in today's scrum to see if Nosh is ok with the lowering of the TPS.

**Comment by Neeharika-Sompalli** [comment]
Regarding logs:  Agreed, Not sure why logs for some tests are not uploaded to AWS. I will create a new issue for it. 
But the logs can also be found in `circelCI job`-> artifacts section, to investigate this issue. for example : https://app.circleci.com/pipelines/github/hashgraph/hedera-services/6709/workflows/22ca03ef-96b8-47f8-8e3a-a05f1a787fe9/jobs/41742/artifacts

**Comment by ljianghedera** [comment]
Should be fixed by PR: https://github.com/swirlds/swirlds-platform-regression/pull/677

**Comment by Neeharika-Sompalli** [comment]
Closed by swirlds/swirlds-platform-regression#677
